RecoEcal/EgammaCoreTools/src/Mustache.cc

```diff
@@ -68,25 +68,25 @@ namespace reco {
     bool inDynamicDPhiWindow(
         const float seedEta, const float seedPhi, const float ClustE, const float ClusEta, const float ClusPhi) {
       // from Rishi's fits 06 June 2013 in log base 10
```

Comment:
The new parameters are not taken from the Rishi's fit  of more than 7 years ago, I imagine: either you update the comment, or better to remove it

Reply:
Updated the comment to add the correct reference.
Thanks,
Rajdeep